lib/u3d/commands.rb

```diff
@@ -328,8 +328,11 @@ def enforce_setup_coherence(packages, options, unity, definition)
             options[:installation_path] ||= unity.root_path if definition.os == :win
           end
           if unity.packages
-            unity.packages.each do |pack|
-              UI.important "Ignoring #{pack} module, it is already installed" if packages.delete(pack)
+            installed_packages = unity.packages
+            packages.each do |pack|
+              if installed_packages.include?(pack) || installed_packages.any? { |installed| !package_aliases[pack].nil? && package_aliases[pack].include?(installed) }
```

Comment:
feel free to move this whole logic into a method "package_installed?" or similar.